src/dev/riscv/LupV.py

```diff
@@ -171,6 +177,12 @@ def attachOffChipIO(self, bus):
             device.pio = bus.mem_side_ports
 
     def generateDeviceTree(self, state):
+        chosen_node = FdtNode("chosen")
+        chosen_node.append(FdtPropertyWords("kernel-start", [0x80000000]))
+        chosen_node.append(FdtPropertyStrings("stdout-path",
+                            "/soc/doesnotexist@0"))
+        yield chosen_node
+
```

Comment:
The chosen node is unrelated to the RNG component. This should be part of another PR.